client/python/openlineage/client/generated/catalog_dataset.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018-2025 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import attr
+from openlineage.client.generated.base import DatasetFacet
+
+
+@attr.define
+class CatalogDatasetFacet(DatasetFacet):
```

Comment:
Java classes are generated while building .jar, am I right?

Reply:
Yes.

We could move this to runtime and add pre-commit checks, to make those changes more visible in PRs.

Reply:
This is just a question :)